utils.py

```diff
@@ -234,9 +301,9 @@ def remove_and_create_folder(folder_path: str, toCreate: bool) -> None:
         os.makedirs(folder_path)
 
 
-def check_if_data_available(batches: list, url: str, apikey: str) -> list:
+def check_if_data_available(batches: list, url: str) -> list:
```

Comment:
I would expect the function to return a bool given it's name.
i think something like `get_new_data` or `get_new_batches`  would make more sense. What do you think ?